### PR TITLE
gh-3201 switch between block explorer and owner details

### PR DESCRIPTION
**Pull request by biafra23**
Handles #3201

social account | other accounts
--------------- | ---------------
image | image

Changes proposed in this pull request:
- As per Tanyas request, the whole row is tappable and leads to the key details page

### CODE REVIEW

File: Multisig/UI/UI Library/Views/AddressInfoView.swift [hunk 1]
```diff
@@ -15,6 +15,7 @@ class AddressInfoView: UINibView {
     @IBOutlet private weak var addressLabel: UILabel!
     @IBOutlet private weak var copyButton: UIButton!
     @IBOutlet private var detailButton: UIButton!
```
Comment (biafra23) [inline Multisig/UI/UI Library/Views/AddressInfoView.swift]: I tried using setImage() before, but there was no image visible. See previous commits. So I made an extra whole button. :-/

**Comment by DmitryBespalov** [comment]
>     The arrow is a bit tough to hit. I assumed we wanted to keep the copy key functionality. If that is not necessary the whole row can be tappable to go to the key details.

@TanyaEfremova does that sounds OK? I have a doubt that if there's an arrow on the row, user needs to click the arrow to open key details instead of just tapping anywhere in the row

File: Multisig/UI/UI Library/Views/AddressInfoView.swift [hunk 2]
```diff
@@ -190,9 +208,14 @@ class AddressInfoView: UINibView {
...
-        if let url = sender.browseURL {
-            openInSafari(url)
+        if sender.showDetailNavigation,
+           let keyInfo = try? KeyInfo.firstKey(address: sender.address) {
+                let vc = OwnerKeyDetailsViewController(keyInfo: keyInfo)
+                show(vc, sender: self)
+        } else {
+            if let url = sender.browseURL {
+                openInSafari(url)
+            }
```
Comment (DmitryBespalov) [inline Multisig/UI/UI Library/Views/AddressInfoView.swift]: this is not how this should be implemented. Instead, we need to use a simple table view row with accessory input, and put the address there. We should have examples of such thing in several places in the interface.

**Comment by TanyaEfremova** [comment]
@DmitryBespalov @biafra23 I agree, that the whole row can be tapped, as in this case going to the owner details would be more important than copying the address and this way it's more accessible.

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**